demo/ocs/config/run-ocs

```diff
@@ -2,14 +2,29 @@
 #
 # Copyright 2020 Intel Corporation
 # SPDX-License-Identifier: Apache 2.0
-#
-# Specify the proxy here. For example, if the http proxy is 'http://myproxy.com:900',
-# then http_proxy_host would be 'myproxy.com' and http_proxy_port would be '900'.
-# If no proxies are needed, leave the fields empty.
-http_proxy_host=
-http_proxy_port=
-https_proxy_host=
-https_proxy_port=
+
+SYSTEM_PROPERTIES="-Dserver.port=${SERVER_PORT}"
+SYSTEM_PROPERTIES+=" -Dlogging.config=${LOGGING_CONFIG}"
+SYSTEM_PROPERTIES+=" -Dapplication.version=${APPLICATION_VERSION}"
+
+FILE_SYSTEM_PROPERTIES="-Dfs.root.dir=${FS_ROOT_DIR} -Dfs.owner.keystore=${FS_OWNER_KEYSTORE}"
+FILE_SYSTEM_PROPERTIES+=" -Dfs.owner.keystore-password=${FS_OWNER_KEYSTORE_PASSWORD}"
+FILE_SYSTEM_PROPERTIES+=" -Dfs.devices.dir=${FS_DEVICES_DIR} -Dfs.values.dir=${FS_VALUES_DIR}"
+
+TO_PROPERTIES="-Dto0.rest.api=${TO0_REST_API}"
+TO_PROPERTIES+=" -Dto0.waitseconds=${TO0_WAITSECONDS}"
+TO_PROPERTIES+=" -Dto2.credential-reuse.enabled=${TO2_CREDENTIAL_REUSE_ENABLED}"
```

Comment:
missing entry for to0.scheduler.interval

Reply:
Added.